dev/projection/script-assert.js

```diff
@@ -13,6 +13,15 @@ function showError(element, msg) {
     }
 }
 
+window.ProjectionFrames = []
+window.MotionDebug = {
```

Comment:
Should we prefix this with a motion-specific prefix?

Reply:
Can't this script be TypeScript, btw?

Reply:
It could be TypeScript but it would just add more overhead to what is meant to be a simple test script. Likewise the `ProjectionFrames` and `MotionDebug` vars are just read in the test files, rather than something that will be used by Motion